### ISSUE TITLE
MTSAC Max Path Length Incorrect

### ISSUE THREAD

**Issue by eholly1** [rlworkgroup/garage#1903]
When running MT50 with SAC using the following command:
`make run-headless RUN_CMD="python examples/torch/mtsac_metaworld_mt50.py`

Got this error
```
2020-08-15 02:07:45 | [mtsac_metaworld_mt50] Setting seed to 1
^T2020-08-15 02:09:26 | [mtsac_metaworld_mt50] Obtaining samples...
Traceback (most recent call last):
  File "examples/torch/mtsac_metaworld_mt50.py", line 103, in <module>
    mtsac_metaworld_mt50()
  File "/home/eholly/venv/lib/python3.5/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/eholly/venv/lib/python3.5/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/eholly/venv/lib/python3.5/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/eholly/venv/lib/python3.5/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/eholly/venv/lib/python3.5/site-packages/garage/experiment/experiment.py", line 553, in __call__
    result = self.function(ctxt, **kwargs)
  File "examples/torch/mtsac_metaworld_mt50.py", line 100, in mtsac_metaworld_mt50
    runner.train(n_epochs=epochs, batch_size=batch_size)
  File "/home/eholly/venv/lib/python3.5/site-packages/garage/experiment/local_runner.py", line 485, in train
    average_return = self._algo.train(self)
  File "/home/eholly/venv/lib/python3.5/site-packages/garage/torch/algos/sac.py", line 206, in train
    last_return = self._evaluate_policy(runner.step_itr)
  File "/home/eholly/venv/lib/python3.5/site-packages/garage/torch/algos/mtsac.py", line 189, in _evaluate_policy
    num_trajs=self._num_evaluation_trajectories))
  File "/home/eholly/venv/lib/python3.5/site-packages/garage/np/_functions.py", line 62, in obtain_evaluation_samples
    deterministic=True)
  File "/home/eholly/venv/lib/python3.5/site-packages/garage/sampler/utils.py", line 66, in rollout
    next_o, r, d, env_info = env.step(a)
  File "/home/eholly/venv/lib/python3.5/site-packages/garage/envs/multi_env_wrapper.py", line 231, in step
    obs, reward, done, info = self.env.step(action)
  File "/home/eholly/venv/lib/python3.5/site-packages/garage/envs/garage_env.py", line 154, in step
    observation, reward, done, info = self.env.step(action)
  File "/home/eholly/venv/lib/python3.5/site-packages/garage/envs/normalized_env.py", line 153, in step
    next_obs, reward, done, info = self.env.step(scaled_action)
  File "/home/eholly/venv/lib/python3.5/site-packages/garage/envs/garage_env.py", line 154, in stpe
    observation, reward, done, info = self.env.step(action)
  File "/home/eholly/venv/lib/python3.5/site-packages/metaworld/envs/mujoco/multitask_env.py", line 161, in step
    obs, reward, done, info = self.active_env.step(action)
  File "/home/eholly/venv/lib/python3.5/site-packages/metaworld/envs/mujoco/sawyer_xyz/sawyer_plate_slide_back_side.py", line 124, in step
    self.do_simulation([action[-1], -action[-1]])
  File "/home/eholly/venv/lib/python3.5/site-packages/metaworld/envs/mujoco/mujoco_env.py", line 118, in do_simulation
    raise ValueError('Maximum path length allowed by the benchmark has been exceeded')
ValueError: Maximum path length allowed by the benchmark has been exceeded
Makefile:187: recipe for target 'run-headless' failed
make: *** [run-headless] Error 1
```

The fix was as follows:
```
diff --git a/examples/torch/mtsac_metaworld_mt50.py b/examples/torch/mtsac_metaworld_mt50.py
index 224c08ad..1259de15 100755
--- a/examples/torch/mtsac_metaworld_mt50.py
+++ b/examples/torch/mtsac_metaworld_mt50.py
@@ -84,7 +84,7 @@ def mtsac_metaworld_mt50(ctxt=None, seed=1, use_gpu=False, _gpu=0):
                   qf1=qf1,
                   qf2=qf2,
                   gradient_steps_per_itr=150,
-                  max_path_length=250,
+                  max_path_length=150,
                   eval_env=mt50_test_envs,
                   env_spec=mt50_train_envs.spec,
                   num_tasks=10,
diff --git a/src/garage/torch/algos/mtsac.py b/src/garage/torch/algos/mtsac.py
index a25318be..f55aaa8b 100644
--- a/src/garage/torch/algos/mtsac.py
+++ b/src/garage/torch/algos/mtsac.py
@@ -186,6 +186,7 @@ class MTSAC(SAC):
                 obtain_evaluation_samples(
                     self.policy,
                     self._eval_env,
+                    max_path_length=self.max_path_length,
                     num_trajs=self._num_evaluation_trajectories))
         eval_trajs = TrajectoryBatch.concatenate(*eval_trajs)
         last_return = log_multitask_performance(epoch, eval_trajs,
```

**Comment by avnishn** [comment]
Thank you for catching this @eholly1! We're keeping track of your second fix right now in #1853 
and I'll go ahead and open a pr for your first fix.

**Comment by avnishn** [comment]
tracking your first change via #1905
